docs/samples/llmisvc/single-node-gpu/llm-inference-service-pd-qwen2-7b-gpu-private.yaml

```diff
@@ -0,0 +1,163 @@
+apiVersion: serving.kserve.io/v1alpha1
+kind: LLMInferenceService
+metadata:
+  name: qwen2-7b-instruct-pd
+  annotations:
+    k8s.v1.cni.cncf.io/networks: roce-p2
+spec:
+  replicas: 1
+  model:
+    uri: hf://Qwen/Qwen2.5-7B-Instruct
+    name: Qwen/Qwen2.5-7B-Instruct
+  router:
+    scheduler:
+      template:
+        containers:
+        # ALWAYS DO PD IN THIS EXAMPLE (THRESHOLD 0)
+        - name: main
+          args:
+          - -v=4 # Debug level
+          - --poolName
+          - "{{ ChildName .ObjectMeta.Name `-inference-pool` }}"
+          - --poolNamespace
+          - "{{ .ObjectMeta.Namespace }}"
+          - --zap-encoder
+          - json
+          - --grpcPort
+          - "9002"
+          - --grpcHealthPort
+          - "9003"
+          - --secureServing
+          - --modelServerMetricsScheme
+          - https
+          - --modelServerMetricsHttpsInsecureSkipVerify
+          - --certPath
+          - /etc/ssl/certs
+          - --configText
+          - |2
```

Comment:
That's some nice YAML-fu :) Would adding it to samples make sense? Maybe not replacing it here for the sake of a self-contained sample.

Reply:
I think this use will need to be streamlined a little bit more in the API (somehow)

Reply:
Got it, I think it makes sense.